ui/redux/actions/app.js

```diff
@@ -45,13 +45,15 @@ import { selectDaemonSettings, selectClientSetting } from 'redux/selectors/setti
 import { selectUser, selectUserVerifiedEmail } from 'redux/selectors/user';
 import { doSetPrefsReady, doPreferenceGet, doPopulateSharedUserState, syncInvalidated } from 'redux/actions/sync';
 import { doAuthenticate } from 'redux/actions/user';
-import { lbrySettings as config, version as appVersion } from 'package.json';
+import p from 'package.json';
 import analytics from 'analytics';
 import { doSignOutCleanup } from 'util/saved-passwords';
 import { LocalStorage, LS } from 'util/storage';
 import { doNotificationSocketConnect } from 'redux/actions/websocket';
 import { stringifyServerParam, shouldSetSetting } from 'util/sync-settings';
 
+const { lbrySettings: config, version: appVersion } = p;
```

Comment:
Small change due to webpack being more strict about destructuring from a module that only has a default export vs named exports.

Reply:
Rereading the error for a screenshot, it says that soon only default will be available:
![image](https://user-images.githubusercontent.com/128739/180585650-3553db0c-e398-416e-a268-b2c26e8bc32a.png)